src/backend/AbsSecBlock.c

```diff
@@ -347,7 +347,8 @@ int BSL_AbsSecBlock_DecodeFromCBOR(BSL_AbsSecBlock_t *self, BSL_Data_t encoded_c
         QCBORDecode_GetUInt64(&asbdec, &tgt_num);
         BSL_LOG_DEBUG("got tgt %" PRIu64 "", tgt_num);
         uint64_list_push_back(self->targets, tgt_num);
-        assert(quit++ < 20);
+        // FIXME
```

Comment:
Use "TODO" and it will be picked up by doxygen. That can also be adjusted to match FIXME.